content/pages/about.md

```diff
@@ -1,7 +1,9 @@
 # About
```

Comment:
We thought it a little strange that you get a "dummy" page, added a public explainer as well as instructions in the Readme.